pkg/cron/cron.go

```diff
@@ -96,7 +96,7 @@ func notifyOnErr(err error, backoffDelay time.Duration) {
 // RequestCall - ZMS call for update crons
 func (c *Cron) requestCall() error {
 	master := false
-	domains, etag, err := c.zmsClient.GetSignedDomains("", "true", "", &master, c.etag)
+	domains, etag, err := c.zmsClient.GetSignedDomains("", "true", "", &master,&master, c.etag)
```

Comment:
Do we need to pass in the `&master` twice ? what is it being used for ?

Reply:
Hi @bhaskaramVaranasi,
Please find the attached screenshot of the function `GetSignedDomains()`.
The `conditions` field has been added as a parameter to the updated package. So to accomodate this, we are sending it as a boolean set to `false`.

![Screenshot 2024-06-03 at 10 17 01 AM](https://github.com/AthenZ/k8s-athenz-syncer/assets/65297307/bd940d12-e4eb-409f-8267-4ec2bebff695)